src/blocks/ui/components/variants.js

```diff
@@ -85,17 +28,16 @@ function Variants( props ) {
 	}
 
 	const fallback = ( variants && variants.find( variant => variant.fallback ) ) || {};
-	const fallbackData = analytics ? ( analytics.audiences.find( audience => audience.id === 0 ) || defaultVariantAnalytics ) : null;
+	const fallbackData = analytics ? ( analytics.variants.find( variant => variant.id === 0 ) || defaultVariantAnalytics ) : null;
 
 	return (
 		<VariantsList>
-			<h2>{ __( 'Block Variants', 'altis-analytics' ) }</h2>
 			{ variants.map( variant => {
-				const data = ( analytics && analytics.audiences.find( audience => audience.id === ( variant.audience && variant.audience.id ) || 0 ) ) || defaultVariantAnalytics;
+				const data = ( analytics && analytics.variants.find( v => v.id === variant.id || 0 ) ) || defaultVariantAnalytics;
 				return (
 					<div className={ `altis-analytics-block-variant ${ variant.fallback ? 'altis-analytics-block-variant--fallback' : '' }` }>
 						<div className="altis-analytics-block-variant__header">
-							<h3>{ variant.fallback ? __( 'Fallback', 'altis-analytics' ) : decodeEntities( variant.audience.title ) }</h3>
+							<h3>{ ! variant.title && variant.fallback ? __( 'Fallback', 'altis-analytics' ) : decodeEntities( variant.title ) }</h3>
```

Comment:
Better to account for translated content as well.

```suggestion
							<h3>{ decodeEntities( ! variant.title && variant.fallback ? __( 'Fallback', 'altis-analytics' ) : variant.title ) }</h3>
```

Reply:
Ah nice catch!